src/web/package.json

```diff
@@ -12,7 +12,6 @@
     "@fontsource/spartan": "^4.2.2",
```

Comment:
There are more changes to this file than I'd expect.  Can you confirm they are all needed?

Reply:
A few I can confirm. I was wondering myself too. I'm cleaning it up now.